docs/guides/using_black_with_other_tools.md

```diff
@@ -145,7 +145,7 @@ There are a few deviations that cause incompatibilities with _Black_.
 
 ```
 max-line-length = 88
-extend-ignore = E203, E704
+extend-ignore = E203, E701, E704
```

Comment:
The paragraph below should explain why E701 and E704 should be ignored too.

Reply:
Also I feel like we could be clearer about which paragraph refers to which option. Perhaps a bulleted list with the relevant error code at the beginning of each entry.